ui/page/exchange/create_order_page.go

```diff
@@ -1116,9 +1127,11 @@ func (pg *CreateOrderPage) getExchangeRateInfo() error {
 	fromCur := pg.fromCurrency.String()
 	toCur := pg.toCurrency.String()
 	params := api.ExchangeRateRequest{
-		From:   fromCur,
-		To:     toCur,
-		Amount: libwallet.DefaultRateRequestAmount, // amount needs to be greater than 0 to get the exchange rate
+		From:        fromCur,
+		FromNetwork: getNetwork(fromCur, pg.instantCurrencies),
+		To:          toCur,
+		ToNetwork:   getNetwork(toCur, pg.instantCurrencies),
+		Amount:      libwallet.RateRequest(fromCur), // amount needs to be greater than 0 to get the exchange rate
```

Comment:
why are we overriding the amount inputted by the user?

Reply:
oh i see this is just ti get the exchange rate

Reply:
Yes